bin/02_aws.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+sudo sed -i 's/PasswordAuthentication no/PasswordAuthentication yes/' /etc/ssh/sshd_config
+
+sudo tee /etc/update-motd.d/00-header <<EOF
```

Comment:
`EOF` instead of EOL, and also dropped the `/dev/null` redirect (it is more clear what is happening during provisioning, because the results are echoed back to `packer`. This is nice because this block should template the expanded ${QIIME2_RELEASE} var into the tee-ed file contents (e.g. `2017.4`).